Documentation/Getting-Started/glossary.md

```diff
@@ -98,12 +98,6 @@ A stretched cluster is a deployment model in which two datacenters with low late
 
 The [Rook toolbox](../Troubleshooting/ceph-toolbox.md) is a container with common tools used for rook debugging and testing.
 
-## Koor
-
-### Koor Storage Distribution (KSD)
-
-Koor Storage Distribution (KSD) is a product based on Rook Ceph that makes running and maintaining Ceph storage on Kubernetes easier.
-
```

Comment:
We don't ant this anymore?

Reply:
I don't know why the git merge with `--ours` flag and checking out the `Documentation/` dir to our state is not preventing this.. I fixed it now.